src/art/__init__.py

```diff
@@ -10,18 +11,18 @@
     import unsloth  # type: ignore
 
 from . import dev
+from .api import API
 from .gather import gather_trajectories, gather_trajectory_groups
 from .model import Model, TrainableModel
 from .trajectories import Trajectory, TrajectoryGroup
 from .types import Messages, MessagesAndChoices, Tools, TrainConfig
-from .local import LocalAPI
 from .utils import retry
 
 __all__ = [
     "dev",
     "gather_trajectories",
     "gather_trajectory_groups",
-    "LocalAPI",
```

Comment:
Removing the `LocalAPI` export is a breaking change.

Reply:
Cool, can upgrade the major version!